testing-library/src/testing/index.ts

```diff
@@ -13,6 +13,7 @@ declare let require: any
 export const test: TestDefinitionFunction = DEBUG
   ? /* @__PURE__ */ createTestFunction()
   : /* @__PURE__ */ () => {}
+export const skipTest: TestDefinitionFunction = () => {}
```

Comment:
is this duplicated? I remember this had been added

Reply:
fixed